doc/rados/operations/crush-map-edits.rst

```diff
@@ -32,193 +33,199 @@ Pool Values`_.
-crush rule.
+A device can also have a *device class* associated with it: for example,
+``hdd`` or ``ssd``. Device classes allow devices to be targeted by CRUSH rules.
+
+To see a list of devices, run the following command:
 
 .. prompt:: bash #
 
-	devices
+    devices
+
+The output of this command takes the following form:
 
 ::
 
-	device {num} {osd.name} [class {class}]
+    device {num} {osd.name} [class {class}]
 
 For example:
 
 .. prompt:: bash #
 
-	devices
+    devices
 
 ::
 
-	device 0 osd.0 class ssd
-	device 1 osd.1 class hdd
-	device 2 osd.2
-	device 3 osd.3
+    device 0 osd.0 class ssd
+    device 1 osd.1 class hdd
+    device 2 osd.2
+    device 3 osd.3
+
+In most cases, each device maps to a corresponding ``ceph-osd`` daemon. This
+daemon might map to a single storage device, a pair of devices (for example,
+one for data and one for a journal or metadata), or in some cases a small RAID
+device.
 
-In most cases, each device maps to a single ``ceph-osd`` daemon.  This
-is normally a single storage device, a pair of devices (for example,
-one for data and one for a journal or metadata), or in some cases a
-small RAID device.
 
-CRUSH Map Bucket Types
+CRUSH-Map Bucket Types
 ----------------------
 
-The second list in the CRUSH map defines 'bucket' types. Buckets facilitate
-a hierarchy of nodes and leaves. Node (or non-leaf) buckets typically represent
-physical locations in a hierarchy. Nodes aggregate other nodes or leaves.
-Leaf buckets represent ``ceph-osd`` daemons and their corresponding storage
-media.
+The second list in the CRUSH map defines 'bucket' types. Buckets facilitate a
+hierarchy of nodes and leaves. Node buckets (also known as non-leaf buckets)
+typically represent physical locations in a hierarchy. Nodes aggregate other
+nodes or leaves.  Leaf buckets represent ``ceph-osd`` daemons and their
+corresponding storage media.
 
-.. tip:: The term "bucket" used in the context of CRUSH means a node in
-   the hierarchy, i.e. a location or a piece of physical hardware. It
-   is a different concept from the term "bucket" when used in the
-   context of RADOS Gateway APIs.
+.. tip:: In the context of CRUSH, the term "bucket" is used to refer to
+   a node in the hierarchy (that is, to a location or a piece of physical
+   hardware). In the context of RADOS Gateway APIs, however, the term
+   "bucket" has a different meaning.
 
-To add a bucket type to the CRUSH map, create a new line under your list of
+To add a bucket type to the CRUSH map, create a new line under the list of
 bucket types. Enter ``type`` followed by a unique numeric ID and a bucket name.
-By convention, there is one leaf bucket and it is ``type 0``;  however, you may
-give it any name you like (e.g., osd, disk, drive, storage)::
+By convention, there is exactly one leaf bucket and it is ``type 0``; however,
+you may give the leaf bucket any name you like (for example, ``osd``, ``disk``,
```

Comment:
@zdover23 - This is a case where you tend to prefer a colon after _for example_.